reconcile/saas_file_owners.py

```diff
@@ -226,6 +250,14 @@ def run(dry_run, gitlab_project_id=None, gitlab_merge_request_id=None,
     current_state = baseline['state']
     desired_state = collect_state()
     diffs = [s for s in desired_state if s not in current_state]
+
+    compare_diffs = collect_compare_diffs(current_state, desired_state)
+    compare_diffs_comment_body = '\n'.join([f'- {d}' for d in compare_diffs])
```

Comment:
nit: I guess you can test on `compare_diffs` and join inside once you know you have something to join 😬

Reply:
done!